src/tags/features/offerwall-dynamic-ui/tag.html

```diff
@@ -15,4 +15,6 @@
  -->
 
 <!-- Tagsmith, Offerwall dynamic UI, v1.0.0 -->
-<% script %>
+<div id="__tagsmith_offerwallDynamicUi">
```

Comment:
QQ: So you wrapped the script with div, is this because you wanted to remove the element when the feature is not active? (I'm seeing offerwall-dynamic-ui/script.js L34~)

Reply:
Yes, that's correct!